features/steps/steps.rb

```diff
@@ -172,3 +170,42 @@ def get_version_number(file_path)
   Maze.check.not_include(run_output, "[INFO]")
   Maze.check.not_include(run_output, "[DEBUG]")
 end
+
+Given(/^I package the bugsnag\-cli$/) do
+  @output = `npm pack`
+  @bugsnag_cli_package_path = "#{Dir.pwd}/#{@output.strip}"
+end
+
+And(/^I init a new nodejs project$/) do
+  @base_dir = Dir.pwd
+  # Make a new directory in feature/cli/fixtures
+  Dir.mkdir("#{Dir.pwd}/features/cli/fixture/")
+  # Change directory to new directory
+  Dir.chdir("#{Dir.pwd}/features/cli/fixture/")
+  @output = `npm init -y`
+end
+
+Then(/^I install the bugsnag\-cli via npm in the new project$/) do
+  @output = `npm install #{@bugsnag_cli_package_path}`
+end
+
+Then(/^I install the bugsnag\-cli via yarn in the new project$/) do
+  @output = `yarn add #{@bugsnag_cli_package_path}`
+end
+
+Then(/^I install the bugsnag\-cli via pnpm in the new project$/) do
+  @output = `pnpm add #{@bugsnag_cli_package_path}`
+end
+
+Then('the node_modules bin directory should contain {string}') do |arg|
```

Comment:
Suggest generalising this step a little too - the test would be equally readable:

```suggestion
Then('the {string} directory should contain {string}') do |arg|
```